homeassistant/components/jellyfin/config_flow.py

```diff
@@ -63,11 +65,22 @@ async def async_step_user(
                 errors["base"] = "unknown"
                 _LOGGER.exception(ex)
             else:
-                await self.async_set_unique_id(userid)
+                entry_title = user_input[CONF_URL]
+
+                server_info: dict[str, Any] = connect_result["Servers"][0]
+
+                if (
+                    server_info
+                    and "Name" in server_info
+                    and len(server_info["Name"]) > 0
+                ):
```

Comment:
```suggestion
                if server_info.get("Name"):
```

I think this should be fine as `""` is not truthy

Reply:
ah nice! I don't use walrus enough :)